File: engine/engine-model/src/main/scala/tools/packet/remove/RemoveKite.scala
package tools.packet.remove

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoveKite(private var _objectId: Int, private var _animationType: Int) extends PacketInput {
  def objectId: Int = _objectId

  def animationType: Int = _animationType

  override def opcode(): SendOpcode = SendOpcode.REMOVE_KITE
}

File: engine/engine-model/src/main/java/tools/packet/remove/RemoveKite.java
package tools.packet.remove;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoveKite(Integer objectId, Integer animationType) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.REMOVE_KITE;
   }
}